apps/api/views/participants.py

```diff
@@ -89,8 +89,14 @@ def _update_participant_data(request):
     serializer.is_valid(raise_exception=True)
 
     identifier = serializer.data["identifier"]
+
     platform = serializer.data["platform"]
     team = request.team
+
+    if platform == ChannelPlatform.COMMCARE_CONNECT:
+        # CommCare Connect IDs are case insensitive, so we store them in lowercase
+        identifier = identifier.lower()
```

Comment:
nit: instead of doing this here I would push it into the `ChannelPlatform` class:

```python
identifier = ChannelPlatform(platform).normalize_identifier(identifier)
```

Reply:
Updated